ui/components/app/snaps/snap-ui-renderer/test-utils.tsx

```diff
@@ -53,12 +55,15 @@ export function renderInterface(
     onCancel,
     contentBackgroundColor,
     state = {},
+    metamaskState = {},
   }: RenderInterfaceOptions = {},
 ): RenderInterfaceResult {
   const store = configureStore({
     ...mockState,
+    ...metamaskState,
     metamask: {
       ...mockState.metamask,
+      ...metamaskState?.metamask,
```

Comment:
Solid addition 🔥, I had it on my plate to add this as I was going through my refactor test ticket. This should also be spread in the reducer below if I'm not mistaken.